features/google/src/views/explore/explore.jsx

```diff
@@ -99,11 +140,12 @@ const ExploreView = () => {
 
     return (
         <Screen
-            className="explore"
+            className="explore-view"
```

Comment:
This was simply wrong before? Or using a different class?

Reply:
It was wrong.